docs/core/resilience/snippets/http-resilience/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Net;
+using Microsoft.Extensions.DependencyInjection;
+using Polly;
+using Polly.CircuitBreaker;
+using Polly.Retry;
+
+var services = new ServiceCollection();
+
+var builder = services.AddHttpClient<ExampleClient>(
+    configureClient: static client =>
+    {
+        client.BaseAddress = new("https://jsonplaceholder.typicode.com");
+    });
+
+builder.AddStandardResilienceHandler(options =>
+{
+    options.RetryOptions.BackoffType = DelayBackoffType.Exponential;
+});
+
+builder.AddResilienceHandler("CustomPipeline", static builder =>
+{
+    // See: https://www.pollydocs.org/strategies/retry.html
+    builder.AddRetry(new RetryStrategyOptions<HttpResponseMessage>
+    {
+        BackoffType = DelayBackoffType.Exponential,
+        MaxRetryAttempts = 5,
+        UseJitter = true
+    });
+
+    // See: https://www.pollydocs.org/strategies/circuit-breaker.html
+    var options = new CircuitBreakerStrategyOptions<HttpResponseMessage>()
```

Comment:
Do we need to introduce `options` variable? Can't it be inlined?

Reply:
Yeah, I can update that. I was trying to avoid going beyond 85 characters in width, since it's a recommendation of the docs platform to help avoid horizontal scrolling with code blocks.